azerothcore/azerothcore-wotlk
# fix(Core/Items): Removed tradeable flag from items with temp enchants.

Fixed #6325.



## Changes Proposed:
-  Enchanted items are not tradeable among raid members.

## Issues Addressed:

- Closes #6325
- Closes https://github.com/chromiecraft/chromiecraft/issues/843

## Tests Performed:

- Not tested

## How to Test the Changes:

1. Loot a weapon that is tradeable
2. Imbue it with any of the shaman weapon spells
3. Click on "Okay", when the dialogue pops up
4. See if the dialogue pops up again, when a new imbue gets applied
5. See if the weapon is still tradeable with the group members
6. 


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
